### PR TITLE
Infrastructure Changes

### PR DESCRIPTION
**Pull request by vherus**

### CODE REVIEW

File: ci/before_install.sh [hunk 1]
```diff
@@ -12,7 +12,11 @@ elif [[ ${TRAVIS_OS_NAME} == "windows" ]]; then
     echo "Before install: Windows OS"
 elif [[ ${TRAVIS_OS_NAME} == "linux" ]]; then
     echo "Before install: Linux"
+    sudo apt-add-repository 'deb https://dl.winehq.org/wine-builds/ubuntu/ xenial main'
```
Comment (cronicc) [inline ci/before_install.sh]: Use bionic repo and add release keys
```suggestion
    wget -O- -nc https://dl.winehq.org/wine-builds/winehq.key | sudo apt-key add -
    sudo apt-add-repository 'deb https://dl.winehq.org/wine-builds/ubuntu/ bionic main'
```

File: ci/before_install.sh [hunk 2]
```diff
@@ -12,7 +12,11 @@ elif [[ ${TRAVIS_OS_NAME} == "windows" ]]; then
...
+    sudo apt-get update
     sudo apt-get install --no-install-recommends -y icnsutils graphicsmagick xz-utils
+    sudo apt install --install-recommends --allow-unauthenticated winehq-stable
```
Comment (cronicc) [inline ci/before_install.sh]:
```suggestion
    sudo apt install --install-recommends winehq-stable
```

File: ci/before_install.sh (hunk 1 above)
Comment (zennermac) [reply ci/before_install.sh]: Travis is actually using Xenial as the most up to date image even though bionic is specified. So need to change the travis config and obviously amend the wine repo used, we will add to this commit with the relevant changes.

File: ci/before_install.sh (hunk 2 above)
Comment (zennermac) [reply ci/before_install.sh]: This one is fine, I will add this into the commit amendments.

**Pull request merged**